sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -457,7 +458,7 @@ public void init(CosmosClientMetadataCachesSnapshot metadataCachesSnapshot, Func
                 collectionCache);
 
             updateGatewayProxy();
-            clientTelemetry = new ClientTelemetry(null, UUID.randomUUID().toString(),
+            clientTelemetry = new ClientTelemetry(this, null, UUID.randomUUID().toString(),
```

Comment:
Not related to this change, 
since you are here, can we also update the clientId?
 Currently, it is this.clientId = clientIdGenerator.getAndDecrement() ----> can we change to getAndIncrement() (><)

Reply:
Fixed - also fixed the activeClientCounter (which was never decremented) :-)